templates/solutions/open-source-security/cyber-resilience-act/index.html

```diff
@@ -393,7 +393,7 @@ <h2>
   <section class="p-strip is-deep">
     <div class="u-fixed-width">
       <h2>
-        <a href="/contact-us#get-in-touch">Need help with your CRA roadmap? Contact our experts &nbsp;&rsaquo;</a>
+        <a aria-controls="cra-contact-modal">Need help with your CRA roadmap? Contact our experts &nbsp;&rsaquo;</a>
```

Comment:
We [recently](https://github.com/canonical/canonical.com/pull/1637) added this for c.com modals as well
```suggestion
        <a aria-controls="cra-contact-modal" class="js-invoke-modal">Need help with your CRA roadmap? Contact our experts &nbsp;&rsaquo;</a>
```